eng/versioning.targets

```diff
@@ -0,0 +1,8 @@
+<Project>
+
+  <ItemGroup>
+    <SupportedPlatform Condition="'$(TargetPlatformIdentifier)' != '' and
+                                  '$(TargetPlatformIdentifier)' != 'windows'" Include="unix" />
+  </ItemGroup>

```

Comment:
Please move this out directly into Directory.Build.targets. We don't need a separate file just for this one item group.

Reply:
Or this could go into BeforeTargetFrameworkInference.targets.

Reply:
No problem. I was not sure if we needed this in a new file, I try to bring stuff from runtime as similarly as possible.

Reply:
Question: Is the value in Include case sensitive? In runtime we have it as `"Unix"`, but I noticed the tfms use it in all lowercase, so here I changed the Include value to `"unix"`. Does that matter?

Reply:
MSBuild includes / conditions / values ignore case. So it doesn't really matter here. I would probably make it the same as in runtime.